frontend/packages/insights-plugin/__tests__/insights.fixtures.ts

```diff
@@ -0,0 +1,108 @@
+import { PrometheusHealthPopupProps } from '@console/plugin-sdk';
+
+const funcs = () => ({
+  history: {
+    push: jest.fn(),
+  },
+  location: {
+    hash: '',
+  },
+  PrometheusHealthPopupProps: jest.fn(),
+  K8sResourceKind: jest.fn(),
+  mapMetrics: jest.fn(),
+  _isWaitingOrDisabled: jest.fn(),
+  _isError: jest.fn(),
+  useTranslation: jest.fn(),
+});
+
+const propsForInsightsWidget: PrometheusHealthPopupProps = {
```

Comment:
nit-pick: rename `propsForInsightsWidget` just to `fixtures` or `propsInsightsPopup`